public/app/routes/routes.tsx

```diff
@@ -32,6 +32,10 @@ export function getAppRoutes(): RouteDescriptor[] {
           path: '/apps',
           component: () => <NavLandingPage navId="apps" />,
         },
+        {
+          path: '/alerts-and-incidents',
```

Comment:
Are we missing an entry here for `path: '/monitoring'`?

Reply:
yes